packages/office-ui-fabric-react/src/utilities/decorators/withViewport.tsx

```diff
@@ -42,8 +42,16 @@ export function withViewport<TProps extends { viewport?: IViewport }, TState>(
         leading: false
       });
 
-      this._events.on(window, 'resize', this._onAsyncResize);
-      const { skipViewportMeasures } = this.props as IWithViewportProps;
+      if (window.ResizeObserver) {
```

Comment:
I would recommend calling `getWindow` api from `../../Utilities`. There are cases where window might not be available.

Reply:
(perhaps less of a concern since you're doing this in `componentDidMount`, but good for consistency.